BUILDING.md

```diff
@@ -116,6 +116,10 @@ simple as:
 cmake .. -G Ninja
 ```
 
+If you encounter an error about an inability to link an OpenSSL target on MacOS,
+try setting your [PKG_CONFIG_PATH](#setting-pkg_config_path) to point to your
+Homebrew installation.
```

Comment:
Note: below we recommend setting `PKG_CONFIG_PATH` to `PKG_CONFIG_PATH="/opt/homebrew/opt/openssl@3/lib/pkgconfig"`. That'll actually only work on arm macos due to `/opt/homebrew`, and I'm not sure why we are being explicit about `@3` version of openssl there.

A better way might be `PKG_CONFIG_PATH=$(brew --prefix)/opt/openssl/lib/pkgconfig`.

That should work on all macs, intel or arm64, as well as be openssl version agnostic. Obviously, if we do have a hard requirement on the @3 version of openssl, please disregard that part.

Reply:
I am not! Definitely like this suggestion.

Reply:
that command works for me! thanks for changing!